bin/main.rb

```diff
@@ -0,0 +1 @@
+puts 'Hello World'
```

Comment:
**_`#!/usr/bin/env ruby` should be at the top line of the main.rb file._**
**_`main.rb` should be an executable file; [check this link if you need to know how](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/)_**